docs/design.md

```diff
@@ -128,3 +128,28 @@ The `DockerHubRegistry` requires a `user`, `pass`, and `org` field
 to be set inside the `registry` section of the configuration file. The user
 credentials are used to authenticate API queries, and the organization is the
 target org that APBs will be loaded from.
+
+### Partner RHCC Adapter
+
+The `PartnerRhccAdapter` allows the broker to be bootstrapped from the the partner registry. Like the `DockerHubRegistry` adapter, it will retrieve a list of APBs and load their specs.  The `PartnerRhccAdapterner` requires a valid RHN `user`, and `pass` for the API queries.
+
+The below is an example of how the [broker configuration file](../etc/example-config.yaml) may be configured for the `PartnerRhccAdapter`:
```

Comment:
Instead of linking to ```[broker configuration file](../etc/example-config.yaml)``` I think we should say ```broker configmap``` to be clear what needs to be changed.

Reply:
I would actually prefer the link, because the related content is expected to remain up to date.

Reply:
I think I agree with @eriknelson. I would rather point to further documentation on the configuration.